tests/test_pytypes.py

```diff
@@ -205,6 +222,11 @@ def test_constructors():
     }
     inputs = {k.__name__: v for k, v in data.items()}
     expected = {k.__name__: k(v) for k, v in data.items()}
+    if pytest.PY2:  # Similar to the above. See comments above.
```

Comment:
All these will need to be `env.PY2` after #2397.

Reply:
Or, if this goes in first, then @2397 needs to be rebased and modified for this, which is also fine.

Reply:
This wasn't mine either, but I've pushed 441bc04d44e8889488ac9a4c719df1caa1a11a44 directly to `str-bytes-cleanup`!